### PR TITLE
 add amrfinderplus_data_manager

### PR DESCRIPTION
**Pull request by pimarin**
PR with amrfinderplus data manager for amrfinderplus tool or bakta

### CODE REVIEW

**Comment by pimarin** [comment]
?

**Comment by pimarin** [comment]
I'm a bit surprise by the error from the python lint:
` data_managers/data_manager_build_amrfinderplus/data_manager/data_manager_build_amrfinderplus.py:270:6: E999 SyntaxError: invalid syntax`

The python file don't contain 270 lines

File: data_managers/data_manager_build_amrfinderplus/data_manager/data_manager_build_amrfinderplus.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+dest='/home/pierre/test/toto'
+
+
+
+code :
+aller sur le ftp
+aller dans le repo ftp
+login avec anonymous
+lister les fichier
+telecharger tous les fichiers
```
Comment (bernt-matthias) [inline data_managers/data_manager_build_amrfinderplus/data_manager/data_manager_build_amrfinderplus.py:275]: Here is the line 270 you are looking for. This seems to be invalid python.

Comment (pimarin) [reply data_managers/data_manager_build_amrfinderplus/data_manager/data_manager_build_amrfinderplus.py:275]: this file is totally different from my file ... I don't understand why the git changes are not accepted

Comment (pimarin) [reply data_managers/data_manager_build_amrfinderplus/data_manager/data_manager_build_amrfinderplus.py:275]: oky the master branch are not update for this file despite my merge ...

Comment (pimarin) [reply data_managers/data_manager_build_amrfinderplus/data_manager/data_manager_build_amrfinderplus.py]: okay, thank you, I believed it was a good practice to merge my tool branch on the master after update from iuc github.

**Comment by bgruening** [comment]
This one needed a rebase I think.

**Pull request closed without merging**